testapi.pm

```diff
@@ -2283,8 +2283,14 @@ A typical call would look like:
 sub compat_args {    # no:style:signatures
     my ($def_args, $fix_keys) = splice(@_, 0, 2);
     my %ret;
-    for my $key (@{$fix_keys}) {
-        $ret{$key} = shift if (scalar(@_) >= 1 && (!defined($_[0]) || !grep { $_ eq $_[0] } keys(%{$def_args})));
```

Comment:
Not related to the PR, but I never looked closely at this code. It occurred to me that for looking if a key exists in a hash, you don't need grep:
```
# old:
!grep { $_ eq $_[0] } keys(%{$def_args}
# new
! exists $def_args->{$_[0]}
```
That would be a simple optimization.

Reply:
thx, I have added it anyway...